arch/cpu/arm/cortex-m/Makefile.cortex-m

```diff
@@ -15,6 +15,22 @@ CONTIKI_ARM_DIRS += cortex-m
 ### Build syscalls for newlib
 MODULES += os/lib/newlib
 
+ifdef TRUSTZONE_SECURE_BUILD
```

Comment:
This ifdef can be  removed, nothing uses `CMSE_IMPORT_LIB` unless building for TrustZone.